code/game/objects/objs.dm

```diff
@@ -23,6 +23,19 @@
 	// Reagent ID => friendly name
 	var/list/reagents_to_log=list()
 
+	var/on_blueprints = FALSE //Are we visible on the station blueprints at roundstart?

```

Comment:
Sorry but TG's version of these vars is retarded, the one on /vg/ (is it on the holomap, and should it be automatic) is way better.